### PR TITLE
Add Python3.7 to CI tests

### PR DESCRIPTION
**Pull request by alxpy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,15 @@
 sudo: false
 language: python
 python:
-    - "2.7.11"
-    - "3.5"
-    - "3.6"
+  - 2.7.11
+  - 3.5
+  - 3.6
```
Comment (dbader) [inline .travis.yml]: Can you confirm these still get parsed as strings? I remember there was some issue with it ending up as float values and breaking the config and so I added the quotes.

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
```
Comment (dbader) [inline .travis.yml]: Do we need that `matrix` stuff? Can't we just add `"3.7"` to the list above?

File: .travis.yml (hunk 1 above)
Comment (dbader) [reply .travis.yml]: Also I'd rather not change the indent because it clutters up the diff.

File: .travis.yml [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
 install: pip install tox-travis coveralls
 script:
-    - tox
-    - if [ $TRAVIS_TEST_RESULT -eq 0 ]; then coveralls; fi
+  - tox
+  - if [ $TRAVIS_TEST_RESULT -eq 0 ]; then coveralls; fi
```
Comment (dbader) [inline .travis.yml]: Let's keep this at 4 spaces

File: README.rst [hunk 4]
```diff
@@ -25,7 +25,7 @@ Features
 - A simple to use API for scheduling jobs.
 - Very lightweight and no external dependencies.
 - Excellent test coverage.
-- Tested on Python 2.7, 3.5, and 3.6
+- Tested on Python 2.7, 3.5, 3.6 and 3.7
```
Comment (dbader) [inline README.rst]: (nitpicky formatting comment): This removes the Oxford comma before the "and" :P

**Comment by dbader** [comment]
Thanks @alxpy! Left some comments to resolve before we can merge.

File: .travis.yml (hunk 3 above)
Comment (alxpy) [reply .travis.yml]: ok, no problem =)

**Comment by alxpy** [comment]
fixed

**Comment by dbader** [comment]
Perfect, thank you!

**Comment by dbader** [comment]
FYI Turns out this doesn't actually run the tests on 3.7 (plus .travis.yml had a parse error) → https://travis-ci.org/dbader/schedule/jobs/424635644. 

Rolling this back until Travis makes it easy to run tests on 3.7.

**Pull request merged**